control/vehicle_cmd_gate/README.md

```diff
@@ -0,0 +1,62 @@
+# Vehicle Cmd Gate
+
+## Purpose
+
+`vehicle_cmd_gate` is the package to get information from emergency handler and planning module and external controller and send a msg to vehicle.
```

Comment:
```suggestion
`vehicle_cmd_gate` is the package to get information from emergency handler, planning module, external controller, and send a msg to vehicle.
```
Nit: It would be easier to read.